ci-operator/config/redhat-appstudio/managed-gitops/redhat-appstudio-managed-gitops-main.yaml

```diff
@@ -1,6 +1,15 @@
 build_root:
```

Comment:
My suggestionfor your use case is to add a new file named, 

ci-operator/config/redhat-appstudio/managed-gitops/redhat-appstudio-managed-gitops-openshift-ci-testing.yaml

And then copy-paste everything from main branch and change branch name in that file (line 69)

Finally run, 

`make ci-operator-config && make jobs && make prow-config`